### DISCUSSION TITLE
build error

### DISCUSSION THREAD

**Discussion by mud0107** [filecoin-project/lotus#5567] [Storage Provider]
**Describe the bug**

**To Reproduce**
1.Run'...'
   Build: os: mac: 10.15.7, golang: 15.6(15,8)
                     linux (ubuntu 20.xx), golang 15.6
2. See error

# github.com/filecoin-project/lotus/chain/gen
chain/gen/mining.go:154:14: undefined: ffi.CreateZeroSignature
make: *** [lotus] Error 2

**Expected behavior**

**Screenshots**

**Version (run `lotus version`):**

git source last version

**Additional context**

**Comment by nonsense** [comment]
I also have an Ubuntu setup and cannot reproduce your problem.

Did you go through the installation instructions at https://docs.filecoin.io/get-started/lotus/installation/#build-and-install-lotus ?

Could you paste the commands you use to try to build Lotus after you `git clone https://github.com/filecoin-project/lotus` ?
